packages/gatsby/src/services/initialize.ts

```diff
@@ -286,19 +282,15 @@ module.exports = async (args: BootstrapArgs) => {
   activity.end()
 
   activity = reporter.activityTimer(`copy gatsby files`, {
-    parentSpan: bootstrapSpan,
+    parentSpan,
   })
   activity.start()
   const srcDir = `${__dirname}/../../cache-dir`
   const siteDir = cacheDirectory
   const tryRequire = `${__dirname}/../utils/test-require-error.js`
   try {
-    await fs.copy(srcDir, siteDir, {
-      clobber: true,
```

Comment:
Curious: why was this no longer required? Is that a change that should be it's own PR?

Reply:
This was an error surfaced when porting to TypeScript. `clobber` is an invalid option (and is the default behaviour anyway). It's ignored, so doesn't cause any harm, but does cause a TS error.

Reply:
yeah it's now called override but it's true by default